tutorials/quantize.md

```diff
@@ -21,7 +21,9 @@ To reduce the memory requirements further, Lit-GPT supports several quantization
 
 ## `bnb.nf4`
 
-Enabled with [bitsandbyes](https://github.com/TimDettmers/bitsandbytes). Check out the [paper](https://arxiv.org/abs/2305.14314v1) to learn more about how it works.
+Enabled with [bitsandbytes](https://github.com/TimDettmers/bitsandbytes). Check out the [paper](https://arxiv.org/abs/2305.14314v1) to learn more about how it works.
+
+> **Note**: for now `bitsandbytes` only supports `CUDA` device and `Linux` operating system.
```

Comment:
```suggestion
> **Note**: `bitsandbytes` only supports `CUDA` devices and the `Linux` operating system.
```

nit: we don't have any visibility on the timeline. "for now" makes it sound like it might change

Reply:
A little bit of hope doesn't hurt :)
But in all seriousness I agree, it's better to be careful with words.

Though I have a feeling that as soon as PyTorch is shipped with CUDA 12 we will revisit it.